### DISCUSSION TITLE
无法访问搜索GitHub项目

### DISCUSSION THREAD

**Discussion by daxianyu1** [community/community#141719] [Other Feature Feedback, Questions, & Ideas]
你们好，我是来自中国的GitHub用户，但是我访问的过程中出现了问题：Whoa there!
You have exceeded a secondary rate limit.

Please wait a few minutes before you try again;
in some cases this may take up to an hour.

我等待两天也不行，还是会出现这个问题，因为中国无法直接访问GitHub，我只能使用VPN软件翻墙来访问GitHub，由于我频繁的更换VPN国家节点（因为有时候我正在使用的VPN节点无法正常访问才更换节点保证我正常访问GitHub），我需要怎么做才能解除这个问题，中国大陆的IP无法直接访问GitHub。

**Comment by daxianyu1** [comment]
我很确定的我不是机器人

**Comment by bronglil** [comment]
Here are a few key points to solve the GitHub rate limit issue:

- Use a Stable VPN with Fixed IP: Choose a VPN that offers a stable connection with a consistent IP address (e.g., NordVPN or ExpressVPN with dedicated IP) to avoid triggering rate limits from frequent IP changes.

- Use SSH keys or Personal Access Tokens (PAT) to authenticate your GitHub actions, which increases your rate limit and reduces the chance of being blocked.

- Avoid frequent requests like page refreshes, repo pulls, or clones. Use GitHub CLI to interact efficiently with repositories.

- Free or heavily shared VPNs are likely flagged by GitHub. Switch to a premium VPN service for better stability and reduced restrictions.

-  If the issue persists, reach out to GitHub Support to review your account and potentially lift restrictions.

**Comment by he-tui-get-out** [comment]
就是没有登录账号导致的

**Comment by hli988** [comment]
你应该是没有登陆你的Github账号吧？登上去之后，再去使用搜索功能。我看好多其他人也遇到了，都是Log in之后就能收缩了。
一句话，Log In

**Comment by shinybrightstar** [comment]
Hi @daxianyu1, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
